src/pykeen/trackers/base.py

```diff
@@ -48,6 +49,44 @@ def end_run(self, success: bool = True) -> None:
         """
 
 
+class PythonResultTracker(ResultTracker):
```

Comment:
documentation request: how would one go about accessing the result tracker after training with the pipeline?

Reply:
6a06ed0

Reply:
thanks :)